app/src/main/java/pl/targosz/invoicing/services/InvoiceService.java

```diff
@@ -1,40 +1,47 @@
 package pl.targosz.invoicing.services;
 
+import java.io.IOException;
 import java.util.List;
+import java.util.Optional;
 import java.util.UUID;
-import pl.targosz.invoicing.database.Repository;
+import pl.targosz.invoicing.database.InvoiceRepository;
 import pl.targosz.invoicing.model.Invoice;
 
-public class InvoiceService implements Repository {
+public class InvoiceService implements InvoiceRepository {
 
-    private final Repository repository;
+    private final InvoiceRepository invoiceRepository;
 
-    public InvoiceService(Repository repository) {
-        this.repository = repository;
+    public InvoiceService(InvoiceRepository invoiceRepository) {
```

Comment:
@RequiredArgsConstructor

Reply:
i can`t, because i use it in code, if i add @RequiredArgsConstructor the test wouldn`t pass, is there any other way?